src/TextInput.tsx

```diff
@@ -90,6 +91,14 @@ const Wrapper = styled.span<StyledWrapperProps>`
     box-shadow: ${get('shadows.formControl')}, ${get('shadows.formControlFocus')};
   }
 
+  ${props =>
+    props.contrast &&
+    css`
+     background-color: ${get('colors.gray.0')};
+    }
+  `}
```

Comment:
Seems like there's one too many brackets here. It's interesting that it sill works
```suggestion
  ${props =>
    props.contrast &&
    css`
     background-color: ${get('colors.gray.0')};
    `}
```